lib/generators/provider/templates/.travis.yml

```diff
@@ -9,6 +9,7 @@ env:
   - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
   - RUBY_GC_HEAP_INIT_SLOTS=600000
   - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
+bundler_args: --no-deployment
```

Comment:
@Fryguy @NickLaMuro is this needed?
I dont really follow https://github.com/ManageIQ/manageiq-content/pull/136